test/pages.js

```diff
@@ -386,6 +386,42 @@ describe('Pages', function() {
     assert.strictEqual(page.path, `${homeId.replace(':en:published', '')}/another-parent/parent/sibling`);
   });
 
+  it('inferred page relationships are correct', async function() {
```

Comment:
Various sanity checks of the inferred target id and position which should be durable if the pages created by the rest of these tests change.